src/config.ts

```diff
@@ -0,0 +1,8 @@
+export const FIREBASE_CONFIG = {
+  apiKey: 'AIzaSyC4FLU4VkT7kcd55TFcQ0gDtjOcIEaGkqk',
```

Comment:
Can you add some documentation on the restrictions you've set up here? And why it's okay to ignore GitHub warnings about this?
Although on that point -- is this something we could put in a GCP secret store and access with the GCP JavaScript client?

Reply:
Changed this to fetch the config from [Firebase hosting](https://firebase.google.com/docs/hosting/reserved-urls#sdk_auto-configuration). The key is actually [intended to be public](https://firebase.google.com/docs/projects/api-keys). I set this up to include the config statically so we don't need to request it when app is live, and because the local dev doesn't serve the config from this relative path + fetching it at runtime from local dev triggers a CORS error.

Reply:
> however, API keys for Firebase services are OK to include in code or checked-in config files.

I'm convinced! We should document this so folks aren't worried by the kind of email we got about leaked API keys.